orc8r/gateway/c/common/test/test_service_registry.cpp

```diff
@@ -1,37 +1,41 @@
-/*
- * Licensed to the OpenAirInterface (OAI) Software Alliance under one or more
- * contributor license agreements.  See the NOTICE file distributed with
- * this work for additional information regarding copyright ownership.
- * The OpenAirInterface Software Alliance licenses this file to You under
- * the terms found in the LICENSE file in the root of this source tree.
+/**
```

Comment:
I'm assuming replacing this license is intentional because this code was originally written by us anyway?